src/devices/CpuTemperature/CpuTemperature.csproj

```diff
@@ -9,5 +9,12 @@
     <Compile Include="CpuTemperature.cs" />
     <None Include="README.md" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="../HardwareMonitor/HardwareMonitor.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Management" Version="5.0.0-preview.5.20224.13" />

```

Comment:
We don't want to use static versions here, instead we should do what we are doing with with the rest of package dependencies from the framework so that autmoatic infrastructure updates will update the version everytime there is a new one on NuGet.org. See what we do here:

[https://github.com/dotnet/iot/blob/aa2ba37e8eeff3ea1e91c4ba97f83abba637c3ba/eng/Version.Details.xml#L19-L34](https://github.com/dotnet/iot/blob/aa2ba37e8eeff3ea1e91c4ba97f83abba637c3ba/eng/Version.Details.xml#L19-L34)

And here:

[https://github.com/dotnet/iot/blob/aa2ba37e8eeff3ea1e91c4ba97f83abba637c3ba/eng/Versions.props#L7-L10](https://github.com/dotnet/iot/blob/aa2ba37e8eeff3ea1e91c4ba97f83abba637c3ba/eng/Versions.props#L7-L10)

I actually need to update those to point to dotnet/runtime as opposed to dotnet/corefx in order to get the right updates for the versions, so I'll take care of that. Let me know if you want me to take care of the ones you are introducing as well.